libtiny_tui/src/tui.rs

```diff
@@ -371,7 +370,7 @@ impl TUI {
     }
 
     /// Returns index of the new tab if a new tab is created.
-    pub fn new_server_tab(&mut self, serv: &str, alias: Option<String>) -> Option<usize> {
+    pub fn new_server_tab(&mut self, serv: &str, alias: &Option<String>) -> Option<usize> {
```

Comment:
If you revert this change (and the `clone()` below) you won't have to make the other changed I asked about in the previous review.